### ISSUE TITLE
Post-install scripts should not be used to install .desktop files

### ISSUE THREAD

**Issue by forivall** [Aluxian/electron-superkit#9]
Instead, they should be included in the package as files in the appropriate location.

**Comment by aluxian** [comment]
I agree with you. I did change this in [Whatsie](https://github.com/Aluxian/Whatsie) which is based on electron-superkit, but I haven't had the time to update the kit.
